modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -23,6 +23,13 @@ You run a DPDK checkup by performing the following steps:
 * You have installed the OpenShift CLI (`oc`).
 * You have configured the compute nodes to run DPDK applications on VMs with zero packet loss.
 
++
+[WARNING]
+====
+The traffic generator pod used by this checkup requires high privileges.
```

Comment:
Please hint the reader to carefully check the RBAC and SCC they apply and call out that the example pulls in upstream container images that Red Hat is not responsible for.